resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -84,6 +84,7 @@ private[spark] object Constants {
   val DEFAULT_EXECUTOR_CONTAINER_NAME = "spark-kubernetes-executor"
   val MEMORY_OVERHEAD_MIN_MIB = 384L
   val NON_JVM_MEMORY_OVERHEAD_FACTOR = 0.4d
+  val KUBERNETES_EXECUTOR_ENV_REGEX = "[A-Za-z_][A-Za-z0-9_]*"
```

Comment:
I think this belongs below, but don't even think it needs to be a val constant that is separate

Reply:
Thanks for your advice, I have removed the val constant and defined a val in the method.